class-tgm-plugin-activation.php

```diff
@@ -1601,6 +1601,33 @@ protected function is_tgmpa_page() {
 		}
 
 		/**
+		 * Determine if we're on a WP Core installation/upgrade page.
+		 *
+		 * @since 2.x.x
+		 *
+		 * @return boolean True when on a WP Core installation/upgrade page, false otherwise.
+		 */
+		protected function is_core_update_page() {
+			// Current screen is not always available, most notably on the customizer screen.
+			if ( function_exists( 'get_current_screen' ) ) {
+
+				$screen = get_current_screen();
+
+				if ( 'update-core' === $screen->base ) {
+					// Core update screen.
+					return true;
+				} elseif ( 'plugins' === $screen->base && isset( $POST['action'] ) ) {
```

Comment:
No, just the fact that it's set is indication enough that one of the WP bulk update screens will be displayed. `$_POST['action']` is only set when a bulk action is requested. (though I should correct the typo ;-) => done)